modules/apps/object/object-test/src/testFunctional/tests/Object.testcase

```diff
@@ -323,6 +323,47 @@ definition {
 			locator1 = "CreateObject#SELECT_OBJECT_FIELD_IN_TABLE");
 	}
 
+	@description = "LPS-162024 - Verify that an error message is shown when the user enters the wrong value in the confirmation field"
+	@priority = "4"
+	test CanViewErrorMessageWithIncorrectValue {
+		task ("Given an object with an attachment field is created") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 162024",
+				objectName = "CustomObject162024",
+				pluralLabelName = "Custom Objects 162024");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customObjectField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject162024");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject162024");
+		}
+
+		task ("When try delete the object with a incorrect value") {
+			ObjectAdmin.openObjectAdmin();
+
+			Click(
+				key_objectLabel = "Custom Object 162024",
+				locator1 = "ObjectAdmin#OBJECT_KEBAB_MENU");
+
+			MenuItem.clickNoError(menuItem = "Delete");
+
+			SelectFrame(locator1 = "ObjectCustomViews#VIEW_MODAL_CONTENT");
+
+			Type(
+				locator1 = "ObjectAdmin#PLACEHOLDER_DELETE_OBJECT",
+				value1 = "anyone");
+		}
+
+		task ("Then the error message is showed on the modal") {
+			AssertElementPresent(locator1 = "ObjectAdmin#ERROR_MESSAGE_INPUT_DOES_NOT_MATCH");
```

Comment:
You can replace it with Message#WARNING_FEEDBACK :)

Reply:
Done, thanks! ;)